pkg/services/rpcsrv/client_test.go

```diff
@@ -1509,9 +1509,9 @@ func TestClient_IteratorSessions(t *testing.T) {
 		wg.Add(storageItemsCount)
 		check := func(t *testing.T) {
 			set, err := c.TraverseIterator(sID, iID, 1)
+			wg.Done()
 			require.NoError(t, err)
 			require.Equal(t, 1, len(set))
-			wg.Done()
```

Comment:
I can't approve this change, it may lead to the failure after the test end. Why have you moved this wg-related code?

Reply:
What's the behavior if error happens? Won't `wg.Wait()` wait forever then?

Reply:
> Yes it will wait forever. as it does in master now. 

What I mean is that it may panic in case of "normal" error returned from iterator traversal (not in case of bug described in the issue when the test hangs). And it's definitely not the desired behaviour.

> You can just replace require with assert here, I think. 

It will help.